Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 
 [features]
 default = ["rayon", "zstd"]
-bench = ["clap", "num_cpus", "env_logger", "rayon", "zstd"]
```

Comment:
Ok. I thought that this list should just include all the direct dependencies of the `bench` tool. 

I just amend the last commit with the requested changed.